ova/provision.sh

```diff
@@ -25,13 +30,19 @@ echo "PermitRootLogin yes" >> /etc/ssh/sshd_config
 # Dependences
 yum install openssl -y
 
-install_wazuh
-
-elastic_stack_${ELK_MAJOR}
+installPrerequisites
+addWazuhrepo
+installWazuh
+installElasticsearch
+installFilebeat
+installKibana
+checkInstallation
+cleanInstall
 
 rm -rf /vagrant
 
 systemctl stop kibana
+systemctl filebeat kibana
 systemctl stop  elasticsearch
 systemctl stop wazuh-manager
-systemctl stop wazuh-api
\ No newline at end of file
+systemctl stop wazuh-api
```

Comment:
The API must only be stopped when the Wazuh version is less than 4.0.0.

Reply:
Done in 4aaa5aee52acb8013e820ae37570eb51a5d588ec